src/node_mgmt/env_scan.py

```diff
@@ -340,26 +348,21 @@ def serial_scan(self, device=None):
 
         result = []
 
-        for br in SERIAL_SCAN_BAUD_RATES:
-            for slave in SERIAL_SCAN_SLAVE_IDS:
-                for sig in SERIAL_SCAN_SIGNATURES:
-                    test = f"Testing slave ID {slave} for '{sig['name']}' at baud rate {br}"
-                    with ModbusRTUReader(device, slave, br, timeout=1, debug=True) as r:
-                        success = True
-                        for rdg in sig['readings']:
-                            try:
-                                response = r.read(**rdg)
-                                res = f"Got response {response}"
-                                if response is None:
-                                    success = False
-                            except Exception as e:
-                                res = f"Error: {e}"
-                                success = False
-                        if success:
-                            res = res + f"==> SUCCESS: Device '{sig['name']}' present as ID {slave} at baud rate {br}"
-
-                    result.append([test, res])
-
+        for sig in SERIAL_SCAN_SIGNATURES:
+            test = f"Testing slave ID {sig['slave_id']} for {sig['name']} at baud rate 9600"
+            with ModbusRTUReader(device, sig['slave_id'], 9600, timeout=1, debug=True) as r:
+                try:
+                    response = int.from_bytes(r.read(register=sig['readings']['register'],
```

Comment:
Per my comment above, in the new approach I don't see the value of having the `register`, `words`, and `fncode` be part of a `readings` object...can just have them be top-level keys in the signature.